zeebe/atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyMessagingService.java

```diff
@@ -500,6 +501,15 @@ private CompletableFuture<Void> loadServerSslContext() {
     }
   }
 
+  private SslContextBuilder sslContextBuilder() {
```

Comment:
This can be refactored to use the default java `.isBlank`, also could be neater to do a to only have one `SslContextBuilder.forServer` call by refactoring the private key password to either be some non blank string or null as https://netty.io/4.1/api/io/netty/handler/ssl/SslContextBuilder.html#forServer-java.io.File-java.io.File-java.lang.String- states that a null value for the password signifies it is not password protected